recipes/raylib/all/test_package/test_package.c

```diff
@@ -6,7 +6,10 @@ int main(void) {
   Vector3 center = {0, 0, 0};
   float r = 1.0;
   if (CheckCollisionSpheres(center, r, center, r)) {
-    printf("unit sphere collides with itself!\n");
+    printf("Success, unit sphere collides with itself!\n");
```

Comment:
Please, revert it. The idea of test package is testing the package itself, not project features: https://github.com/conan-io/conan-center-index/blob/6e688de99164ae5d7940f8c7c138b36c321dab0a/docs/adding_packages/test_packages.md#minimalist-source-code

Reply:
Do you mean that the real test is to see if `test_package.c` compiles?
In that case I would suggest removing the printf, or keeping the "Success" included in this string.

I think this log message is the last thing that the conan create command prints before exiting, 
so to me it was not clear if the conan create command succeeded or had failed.

Reply:
I mean, revert to the state that's available in master branch. The printf does not matter, but failing because a method of the library returns error it actually does.

> so to me it was not clear if the conan create command succeeded or had failed.

Not true, Conan client shows explicitly when test package fails. Example:

```
ERROR: foo/0.1.0 (test package): Error in test() method, line 26
	self.run(cmd, env="conanrun")
	ConanException: Error 1 while executing
```

Reply:
The failure is explicit, but what about the success?

My first impression was that"unit sphere collides with itself!" was an error.
Then it took me a minute to find out where that error came from, before I realized it's in fact a success.
Or rather, as you say, the log message does not matter at all?

I'll revert the change, but I guess I have a feature request for conan client: print a "Package test passed"

Reply:
removed change

Reply:
Thank you!